src/methods/response.js

```diff
@@ -34,13 +34,20 @@ const parseJsonResponse = function (response, textResponse, responseType) {
   try {
     return JSON.parse(textResponse)
   } catch {
-    throw new TextHTTPError(response, textResponse)
+    throw addFallbackErrorMessage(new TextHTTPError(response, textResponse), textResponse)
   }
 }
 
+const addFallbackErrorMessage = function (error, textResponse) {
+  error.message = error.message || textResponse
```

Comment:
The new version of `node-fetch` makes the error message now empty in some cases.